requirements.txt

```diff
@@ -61,6 +61,8 @@ vdm==0.14
 webencodings==0.5.1       # via html5lib
 weberror==0.13.1          # via pylons
 webhelpers==1.3
+feedgen==0.7.0
```

Comment:
I haven't looked at the other parts, but the new dependencies should be added to `requirements.in` instead. Then use `pip-compile` to re-generate `requirements.txt`. Both `requirements.in` and `requirements.txt` are then committed.